wesnoth-multiplayer.po

```diff
@@ -1792,11 +1778,21 @@ msgid ""
 "Pyrennis have proved astoundingly indestructible both to time and war — that "
 "is, what scattered segments of it were actually completed."
 msgstr ""
+"Když začaly skřetí hordy houfně přicházet na Velký kontinent, pustil se "
+"jeden neohrožený vládce zvaný Železný král Olgor do prvního sjednocování "
+"hašteřivých skřetích klanů. Snažil se podmanit pod nadvládu skřetů celý "
+"kontinent a přikázal postavit obrovskou pevnost Pyrennis jako pomník své "
+"moci. Ale právě tato citadela se stala jeho zkázou. Jeho vazalové, neochotní "
+"dokončit stavbu, dokud nebude předurčené dobytí země dokončeno, se obrátili "
+"proti Olgorovi, zabili ho a způsobili tak rozvolnění svazku skřetů trvající "
+"celá další staletí. Zdi Pyrennisu se ukázaly být jako neuvěřitelně pevné, "
+"odolávající jak času, tak válečnému běsnění. Což vyniká zvlášť ve světle "
+"toho, jak roztroušené části pevnosti byly skutečně dokončeny."
```

Comment:
význam úplně nepobírám, jen mi to **jak roztroušené** zní zavádějícně
```suggestion
"toho, jak byly roztroušené části pevnosti skutečně dokončeny."
```

Reply:
No je to tam složité, nějak jsem to intuitivně nahodil.
Myslím si, že má verze lépe zdůrazňuje to, jak je to úžasné ve srovnání s tou zlomkovitostí.